wpeverest/user-registration
# UR-1575 Fix - Removed userstate from attributes.

### All Submissions:






### Changes proposed in this Pull Request:
Fix for removing user state from attributes in login and my account block.


Closes # .

### How to test the changes in this Pull Request:

1. Add my account and user login block
2. change the user state and save the page
3. The user state should not be saved and the block should act as it is supposed to.

### Types of changes:




### Other information:




### Changelog entry
Bug - Removed user state from being saved as attribute for login and my account block.
